### PR TITLE
Update version: jurplel.qView version 5.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115029&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — APPROVED** [review]

**Comment by stephengillie** [comment]
image

File: manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+- webp
+- xbm
+- xpm
+ReleaseDate: 2022-01-13
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/jurplel/qView/releases/download/5.0/qView-5.0-win32.exe
+  InstallerSha256: 7E95CAE01CA4E3CD9DF8F62E50C79E4F04658EBBC617B5D917D132F93114BA61
+  InstallerSwitches:
```
Comment (stephengillie) [inline manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml:46]:
```suggestion
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    InstallerSwitches:
```

File: manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/jurplel/qView/releases/download/5.0/qView-5.0-win32.exe
+  InstallerSha256: 7E95CAE01CA4E3CD9DF8F62E50C79E4F04658EBBC617B5D917D132F93114BA61
+  InstallerSwitches:
```
Comment (stephengillie) [inline manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml:52]:
```suggestion
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    InstallerSwitches:
```

File: manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
+    Custom: /ALLUSERS
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/jurplel/qView/releases/download/5.0/qView-5.0-win64.exe
+  InstallerSha256: 2D493C49AEE638D5FFBC144C822E88D9F6C17162F502C5D43E6FC83712DE0B29
+  InstallerSwitches:
```
Comment (stephengillie) [inline manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml:58]:
```suggestion
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    InstallerSwitches:
```

File: manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml [hunk 4]
```diff
@@ -1,67 +1,67 @@
...
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/jurplel/qView/releases/download/5.0/qView-5.0-win64.exe
+  InstallerSha256: 2D493C49AEE638D5FFBC144C822E88D9F6C17162F502C5D43E6FC83712DE0B29
```
Comment (stephengillie) [inline manifests/j/jurplel/qView/5.0/jurplel.qView.installer.yaml:63]:
```suggestion
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    InstallerSwitches:
```

**Pull request closed without merging**